doc/source/manual/xtrabackup_sst.rst

```diff
@@ -94,7 +94,7 @@ Used to enable and specify SST encryption mode:
 
    :Values: AES128, AES192, AES256
 
-Used to specify the SST encryption algorithm. It uses the same values as the ``--encryption`` option for XtraBackup (see `here <http://www.percona.com/doc/percona-xtrabackup/2.3/innobackupex/encrypted_backups_innobackupex.html>`_). The ``encrypt-algo`` option is considered only if :option:`encrypt` is set to ``1``.
+Used to specify the SST encryption algorithm. It uses the same values as the ``--encryption`` option for XtraBackup (see `this document <http://www.percona.com/doc/percona-xtrabackup/2.3/innobackupex/encrypted_backups_innobackupex.html>`_). The ``encrypt-algo`` option is considered only if :option:`encrypt` is set to ``1``.
```

Comment:
Just for reference not sure if it is mentioned anywhere.

PXC-5.7.12 can operate with latest version that is 2.4.3 of XB.
PXC-5.7.12 shouldn't be used with older version.